scripts/validate_frontend_lib_imports.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
```

Comment:
This script is run from `make jslint`, and it ensures that source files in `frontend/src/lib` don't import from `frontend/src/app` or `frontend/src/assets`.

It's temporary and will be removed once frontend/src/lib is packaged as its own standalone library.